src/index.js

```diff
@@ -3,6 +3,8 @@ import $ from 'jquery';
 import Backbone from 'backbone';
 import L from 'leaflet';
 import App from './js/index';
+window.jQuery = $
```

Comment:
It's fine, just slightly confusing since if $ exist, then it means that window.jQuery should already be there. Instead of assigning it $, do another import of jquery here and assign it that. That way it's less magic.